docs/about/neon_ecosystem.md

```diff
@@ -98,5 +84,27 @@ Oracles enable the decentralized Web3 ecosystem to access price feeds and other
 
 ### Wallets
 
-Neon EVM is supported by the MetaMask non-custodial wallet.
+Neon EVM is supported by the MetaMask non-custodial wallet. 
+
+> This allows dApp devs to sign deployment transactions and mint tokens. 
+<!-- > Learn more about minting and wrapping tokens with Neon EVM. suggest adding link here as Phantom wallet is invovled for Solana-based mints -->
+
+### Protocols
+
+Neon EVM integrates with the popular WalletConnect protocol that provides a decentralized standard to connect Web3 wallets and dApps.
+
+### Tracing API
+
+More sophisticated than NeonScan, Neon's Tracing API is an extension for Neon Proxy. It assists developers to test, debug, and understand the behavior of their smart contracts on Neon EVM. The API provides a full externality trace on any Neon transactions executed on the Solana blockchain. 
```

Comment:
Why are we comparing NeonScan with Tracing API?

Reply:
✅ removed